mirantis/aws-cloud-controller-manager/templates/cloudconfigmap.yaml

```
@@ -13,5 +13,5 @@ data:
     {{- end }}
 kind: ConfigMap
 metadata:
-  name: {{- include "aws-cloud-config.name" . | nindent 4 }}
+  name: {{- include "aws-cloud-config.name" . }}
```

This is not rendering correctly, you have trimmed extra whitespace, so it will render as
```yaml
name:"aws-cloud-config"
```

Which is incorrect yaml, you should do something like this (as suggested earlier):

```suggestion
  name: {{ include "aws-cloud-config.name" . }}
```

Applicable to other places with the same issue